pkg/apis/messaging/v1beta1/channel_defaults.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2019 The Knative Authors
```

Comment:
Maybe change all to 2020?

Reply:
I'm not really sure what it should be, but because the other PRs doing this didn't change the years, I'll leave this one unchanged too.